### ISSUE TITLE
Error accesing savefile.

### ISSUE THREAD

**Issue by MeesNa12** [C3RV1/UndertaleNDS#33]
It shows caught exception: Error accessing savefile, btw i play using a wood ds flashcart

**Comment by C3RV1** [comment]
That means you are running the game from an emulator which does not support saving UndertaleNDS. This is usually a somewhat recent MelonDS. I recommend running the game from MelonDS v0.9.1.

**Comment by C3RV1** [comment]
If it appears on a cardridge, it means that for some reason the game cannot save to the cardridge.

**Comment by JemzoMaclain** [comment]
Is there a reason that that might be the case? I'm having the same issue on an R4.

**Comment by ru9208** [comment]
> Is there a reason that that might be the case? I'm having the same issue on an R4.

are you using a R4wood or R4com ？for me the wood card cant work even I translate the usable .dsv savefile to .sav and put it back in this card still no save state be loaded.

**Comment by Asy-Z** [comment]
yeah i also face the same thing using my r4 card ewith the twilight menu

**Comment by Asy-Z** [comment]
> yeah i also face the same thing using my r4 card with the twilight menu

maybe anyone here mind sharing an alr created .sav file?

**Comment by CapiYT14** [comment]
The same thing happens to me but with the Twilight Menu.

**Comment by soykhaler** [comment]
Dont working on DSPico with Twilight Menu
